app/configurator/components/filters.tsx

```diff
@@ -782,6 +815,12 @@ const DrawerContent = forwardRef<
           }
           sx={{ typography: "body2" }}
         />
+        <FilterControls
+          selectAll={() => setPendingValues(flatOptions)}
+          selectNone={() => setPendingValues([])}
+          allKeysLength={flatOptions.length}
```

Comment:
I guess there should be a uniqueBy value here otherwise we can count twice values that are multiple times in the tree.